docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx

```diff
@@ -21,7 +21,7 @@ The abstract of the paper is the following:
 ## Tips
 
 - Stable Diffusion XL works especially well with images between 768 and 1024.
-- Stable Diffusion XL output image can be improved by making use of a refiner as shown below
+- Stable Diffusion XL output image can be improved by making use of a refiner as shown below.
 
 ### Available checkpoints:
```

Comment:
are these changes intentional? i'm not familiar with the file format.

Reply:
Which fileformat?

Reply:
mdx files

Reply:
We use mdx files for documentation.